src/prop/minisat/core/Solver.h

```diff
@@ -455,6 +455,26 @@ class Solver {
     // Returns a random integer 0 <= x < size. Seed must never be 0.
     static inline int irand(double& seed, int size) {
         return (int)(drand(seed) * size); }
+
+    // Writes to Trace macro for decision tree tracing
+    static inline std::string dtviewDecisionHelper(size_t level,
+                                                   const Node& node,
+                                                   const char* decisiontype)
+    {
+      std::ostringstream msg;
+      msg << std::string(level - (options::incrementalSolving() ? 1 : 0), '*')
+          << " " << node << " :" << decisiontype << "-DECISION:";
+      return msg.str();
+    }
+
+    static inline std::string dtviewPropagationHeaderHelper(size_t level)
+    {
+      std::ostringstream msg;
+      msg << std::string(level + 1 - (options::incrementalSolving() ? 1 : 0),
+                         '*')
+          << " /Propagations/";
+      return msg.str();
+    }
```

Comment:
This doesn't need to be a member of  the Solver class. You can move the helpers to `Solver.cc` into an anonymous namespace.

Reply:
Done. Thanks for the feedback.